insights/parsers/smartctl.py

```diff
@@ -202,3 +211,51 @@ def parse_attributes(line):
 
         # Delete temporary full line storage
         del self.full_line
+
+
+@parser(Specs.smartctl_l_scterc)
+class SMARTctlSCTERC(CommandParser, dict):
+    """
+    Parser for output of ``smartctl -l scterc`` for each drive in system.
+
+    This stores the SCT ERC (Smart Command Transfer Error Recovery Control) information
+    from the output of `smartctl -l scterc` in the following properties:
+    following properties:
+
+    * ``device`` - the name of the device after /dev/ - e.g. sda
```

Comment:
Ideally this information would also be included in an `Attributes:` section of the docstring.